### ISSUE TITLE
Endorsement should allow three states

### ISSUE THREAD

**Issue by ahahn-gbif** [gbif/registry#119]
The confirmation of endorsement presently allows only for yes/no. This makes tracking of an explicit "No" from the endorsing node confusing, as "no" is also the default value for "pending". We recently have a few cases where endorsement is explicitly refused, because the institution is out of scope, with publishing interests that are not in the GBIF realm. An explicit "no" would make this process more transparent than just deleting the organization, which also happens in case of duplicates etc.

**Comment by MortenHofft** [comment]
I completely agree - i find it too limiting as well.
Moved here from the registry-console as this is a model task firstmost. Once that is in place we can change the UI.

**Comment by mike-podolskiy90** [comment]
Duplicate #209
